python/delete_instuck_progress.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+'''
+deletes file in case it exists
+'''
+
+import os
+import os.path
+from datetime import datetime
+from datetime import date
+
+
+def drop_file(abs_folder, filename):
+    if os.path.isfile(os.path.join(abs_folder, filename)):
+        os.remove(os.path.join(abs_folder, filename))
+        return True
+    return False
+
+
+def drop_outdated_inprogress_file(abs_folder, filename):
```

Comment:
Is it correct that `drop_outdated_inprogress_file` seems not be called at all anywhere (apart of in the tests).